app/parse.py

```
@@ -15,8 +19,130 @@ class Product:
     num_of_reviews: int
 
 
+BASE_URL = "https://webscraper.io/"
+HOME_URL = urljoin(BASE_URL, "test-sites/e-commerce/more/")
+
+
+def get_webdriver() -> webdriver.Chrome:
+    options = Options()
+    options.add_argument("--headless=new")
+    return webdriver.Chrome(options=options)
+
+
+def accept_cookies(driver: webdriver.Chrome) -> None:
+    try:
+        accept_button = driver.find_element(By.CLASS_NAME, "acceptCookies")
+        accept_button.click()
+        time.sleep(0.1)
+    except Exception:
+        pass
```

avoid the bare except + pass antipattern; try to be specific in except